recipes/configs/qwen2/1.5B_full_single_device.yaml

```diff
@@ -51,12 +52,15 @@ batch_size: 2
 epochs: 1
 optimizer:
   _component_: bitsandbytes.optim.PagedAdamW
-  lr: 5e-6
+  lr: 2e-5
+
 optimizer_in_bwd: True
+
 loss:
   _component_: torch.nn.CrossEntropyLoss
+
 max_steps_per_epoch: null
-gradient_accumulation_steps: 16
+gradient_accumulation_steps: 1
```

Comment:
What's the reasoning for bumping this down?

Reply:
opt in bwd